libs/discord/discord.cpp

```diff
@@ -0,0 +1,212 @@
+
+//#include "qcommon/qcommon.h"
+#include "cgame/cg_local.h"
+#include "windows/miniwindows.h"
+#include <time.h>
+#include "discord_game_sdk.h"
+
+#define CLIENT_ID 882369979406753812
+
+using tDiscordCreate = decltype(&DiscordCreate);
+
+static HMODULE discord_sdk;
+static tDiscordCreate pDiscordCreate = nullptr;
+
+struct Application {
+	IDiscordCore* core;
+	IDiscordUserManager* users;
+	IDiscordActivityManager* activities;
+	DiscordUserId user_id;
+};
+
+
+template< size_t size >
+void Q_strncpyz( char (&dest)[size], const char * src )
+{
+	Q_strncpyz( dest, src, size );
+}
+
+
+
+void OnUserUpdated(void* data)
+{
+	Application* app = (Application*)data;
+	DiscordUser user;
+	app->users->get_current_user(app->users, &user);
+	app->user_id = user.id;
+
+	Com_GGPrint("Discord: Hello {}#{}", user.username, user.discriminator);
```

Comment:
Can do deeper discord integration, talk with the overlay, show friendslist or what not, but I wasn't sure what you wanted so just added this to show it.